examples/instructlab-multiphase-configs/README.md

```diff
@@ -0,0 +1,16 @@
+# InstructLab Multi-Phase Training Configurations
+
+This directory contains hardware-specific configurations for running the LAB multi-phase training pipeline with different GPU setups.
+
+## Quick Start
+
+See [`lab_multiphase_configs.ipynb`](./lab_multiphase_configs.ipynb) for optimized training parameters for various hardware configurations including:
+
+- **H200**: 1x, 2x, 4x, 8x GPU configurations
```

Comment:
Is this always single node or multi-node is supported? That would be useful to mention it explicitly.

Reply:
Good point, added a disclaimer to the README